Library/SettingsAccountCellType.swift

```diff
@@ -34,6 +34,10 @@ public enum SettingsAccountCellType: SettingsCellTypeProtocol, CaseIterable {
   case currency
   case currencyPicker
```

Comment:
Returning `button` for all rows should also be fine with `currencyPicker` being removed soon.